tests/cephadm/test_cephadm_bootstrap_with_spec_file.py

```diff
@@ -0,0 +1,84 @@
+from cli.exceptions import ConfigError, ResourceNotFoundError
```

Comment:
Can't we use existing test case - https://github.com/red-hat-storage/cephci/blob/master/tests/cephadm/test_cephadm_bootstrap.py ?

Reply:
Updated